pytorch_blade/src/ltc/disc_compiler/passes/graph_fuser.cpp

```diff
@@ -473,13 +473,17 @@ struct GraphFuser {
       mergeFusionGroups(group, producer->node());
       return group;
     }
-
     Node* merged = mergeNodeIntoGroup(group, producer->node());
     // remaining uses of this producer can occur because we allow
     // fusion in cases where uses remain after the consumer
     // if these exist, re-route them to the version of producer
     // created in FusionGroup
+
     auto producer_outputs = producer->node()->outputs();
+    // DISC: just copy constant node into subgraph, scalar output would be core
+    // dump in this pass, we can remove the no-consumer constant node in EDC
```

Comment:
EDC -> DCE?

Reply:
I removed the patch on `graph_fuser.cpp`, because of it's does not related this PR, `graph_fuser` has some other problems on replaying bert training, but it works well on mnist, I make it works well on bert in the next PR.